react-app/src/app/components/Topbar/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { NavLink } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import style from './styles.scss';
+
+class Topbar extends Component {
+  renderLink = (link, i) => {
+    if ((link.restrict && this.props.logged) || !link.restrict) {
+      return (
+        <NavLink
+          key={i.toString()}
+          to={link.to}
+          exact
+          activeClassName={style.topbarActive}
+          className={style.topbarLink}
+        >
+          <FontAwesomeIcon icon={link.icon} />
+          &nbsp;
+          {link.name}
+        </NavLink>
+      );
+    }
+    return null;
+  };
+
+  render() {
+    const links = this.props.links.map((link, i) => this.renderLink(link, i));
```

Comment:
There is no need to define an arrow function. You can use the function reference directly